test/rest_request_template/travis_correct_auto_table_creation.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "SNOWFLAKE_CONNECTOR_NAME",
+  "config": {
+    "connector.class": "com.snowflake.kafka.connector.SnowflakeSinkConnector",
+    "snowflake.role.name": "TESTROLE_KAFKA",
+    "topics": "SNOWFLAKE_TEST_TOPIC",
+    "tasks.max": "1",
+    "buffer.flush.time": "10",
+    "buffer.count.records": "100",
```

Comment:
Can you also add one more config for topic to table map? Should be fairly easy.

Reply:
You mean add another test with the topic to table map config set?

Reply:
Yes, correct